packages/node_modules/@webex/internal-plugin-dss/test/unit/spec/dss.ts

```diff
@@ -195,6 +195,25 @@ describe("plugin-dss", () => {
         expect(result).to.equal("some return value");
       });
 
+      it("calls _request correctly with entityProviderType", async () => {
```

Comment:
suggestion: Equivalent tests for the other `entityProviderType` would be good, or at least one other to prove this isn't always using `CI_USER` when any `entityProviderType` is provided.